### PR TITLE
Feature/responsiveness

### PR DESCRIPTION
**Pull request by Perezented**
# Description
Styling changes for a more responsive experience. Mobile view added to cityReport

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or  feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for dicussion/feedback

## Has this Been Tested?

- [ ] Yes
- [ ] No
- [ ] Not Necessary

## Checklist
- [x] My code follows the style guidelines of this project
- [x] At least 1 reviewer (2 person team as of 9/8/2020)
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/CompareCities/Plotly.js [hunk 1]
```diff
@@ -285,31 +272,36 @@ export default function Plotly(props) {
...
               <p>Today's Forecast: </p>
-              <p>Clouds Today: </p>
-              <p>Humidity: </p>
-              <p>Pressure: </p>
-              <p>Visibility: </p>
-              <p>Wind direction: </p>
-              <p>Wind Speed: </p>
-            </div>
-            <div className="weather-stat-nums">
               <p>{cityNumber.description}</p>
```
Comment (mtoricruz) [inline src/components/pages/CompareCities/Plotly.js:275]: Simple and effective tweak

**Review by mtoricruz — APPROVED** [review]

File: src/components/common/styles/CityReport.scss [hunk 2]
```diff
@@ -206,3 +206,134 @@
...
+    padding: 3px;
+    .walkscore-num {
+      font-size: 2rem;
+    }
+  }
+  h3 {
+    font-weight: 700;
+    padding: 5%;
+  }
+  #cityNumber:nth-child(2) {
```
Comment (Perezented) [inline src/components/common/styles/CityReport.scss:308]: Specifics on css selection

File: src/components/common/styles/CityReport.scss [hunk 3]
```diff
@@ -206,3 +206,134 @@
...
+    grid-template-columns: 100%;
+  }
+  .cityDisplayPlot {
+    width: 100%;
+    padding: 1px;
+    h1 {
+      font-size: 2rem;
+    }
+  }
+  .cityDisplayTablet {
```
Comment (Perezented) [inline src/components/common/styles/CityReport.scss:229]: class changes on tablet view

File: src/components/pages/CompareCities/Plotly.js [hunk 4]
```diff
@@ -446,10 +436,12 @@ export default function Plotly(props) {
   }
   return (
     <section>
-      <RentPlot thisCityData={thisCityData} />
-      <RentPredictViz rentPredictViz={rentPredict} />
-      <UnemploymentPlot unemployment={unemployment} />
-      {/* <JobIndustryViz jobViz={jobIndustryViz} /> */}
+      <div className="vizs">
```
Comment (Perezented) [inline src/components/pages/CompareCities/Plotly.js:439]: Additional div for the grid view

**Pull request merged**